pkg/commands/kustomizationfile.go

```diff
@@ -100,3 +120,73 @@ func stringInSlice(str string, list []string) bool {
 	}
 	return false
 }
+
+func (mf *kustomizationFile) getFieldComments(content []byte) error {
+	buffer := bytes.NewBuffer(content)
+	var comments [][]byte
+	incomments := false
```

Comment:
`inComments`

Reply:
I found `inComments` is not needed. Removed it.